### PR TITLE
Update NanoMed Plus

### LINKED ISSUE #19088: Nanomed Ejects Items Too Quickly
Compared to the rest of the vendors, the nanomed plus ejects their items much quicker than normal, for some reason. Not sure why that vendor specifically, but it is there

### PR DESCRIPTION
**Pull request by justdie12**
## About the PR

resolve #19088
## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by justdie12** [comment]
#19088 solved

**Comment by lzk228** [comment]
please name your PR correctly, you can write short about changes and then describe them fully in pr description
also you can type `resolve #19088` and the issue will be autoclosed if this pr get merged

**Comment by justdie12** [comment]
> please name your PR correctly, you can write short about changes and then describe them fully in pr description also you can type `resolve #19088` and the issue will be autoclosed if this pr get merged

srry dont know

**Comment by justdie12** [comment]
resolve #19088

**Comment by lzk228** [comment]
> resolve #19088

i meant you should write it in your pr description

**Review by VasilisThePikachu — CHANGES_REQUESTED** [review]
Please explain why this pr is good for the game and add a changelog

File: Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml [hunk 1]
```diff
@@ -791,7 +791,6 @@
     normalState: normal-unshaded
     ejectState: eject-unshaded
     denyState: deny-unshaded
-    ejectDelay: 0.6
```
Comment (UbaserB) [inline Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml:794]: Please change it to 1.9 instead of removing it

**Review by UbaserB — CHANGES_REQUESTED** [review]

**Comment by slarticodefast** [comment]
A 2 second delay for the nanomed would feel very frustrating if you need several items. We had a maintainer discussion about this and would you request you to instead reduce the delays of other vending machines to be more consistent. Keep in mind the long vending sound for machines like the coffe vendor, so they don't overlap.

File: Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml (hunk 1 above)
Comment (UbaserB) [reply Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml:794]: scratch this lol

**Comment by UbaserB** [comment]
You coming back to this?

**Pull request closed without merging**